Sources/Networking/HTTPClient/HTTPRequest.swift

```diff
@@ -138,7 +140,8 @@ extension HTTPRequest.Path {
                 .postOfferForSigning,
                 .postReceiptData,
                 .postSubscriberAttributes,
-                .postAdServicesToken:
+                .postAdServicesToken,
+                .getProductEntitlementMapping:
```

Comment:
@tonidero I enabled ETags for this again after talking to the server people. Once that works there, the SDK will support it.

Reply:
Thanks for letting me know. Will probably revert my PR since there won't be any requests using that right now... We can add it back in the future.